src/backend/marsha/core/models/video.py

```diff
@@ -245,6 +262,108 @@ def get_mediapackage_endpoints(self):
         """Return the mediapackage enspoints info."""
         return self.live_info.get("mediapackage").get("endpoints")
 
+    def get_recording_slices(self):
+        """Return empty array if no recording slices exists."""
+        if not self.recording_slices:
+            return []
+        return self.recording_slices
+
+    @property
+    def is_recording(self):
+        """Whether the video is currently recording."""
+        recording_slices = self.get_recording_slices()
+        if not recording_slices:
+            return False
+        return recording_slices[-1].get("end") is None
```

Comment:
```suggestion
        return recording_slices[-1].get("end") is None
```
if it doesn't contains starts it still return true for instance [{"test":"toto"}] maybe you should check the recording got started

Reply:
I don't know how to manage that… setting a wrong value like the one you mentioned should be blocked.
WDYT @carofun @lunika ?

Reply:
I was more thinking of checking stat the key to start the recording is present as your question is_recording ?
so I would find that more secure to check as well that it contains start